### PR TITLE
Added automatic completion install for pidcat

### PR DESCRIPTION
**Pull request by paour**

### CODE REVIEW

File: Library/Formula/pidcat.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 require "formula"
 
 class Pidcat < Formula
+  depends_on "bash-completion" => :recommended
```
Comment (DomT4) [inline Library/Formula/pidcat.rb]: Is this necessary? Most bash completions can just be installed without any added dependencies. If it actually needs it, the dependency should go above `def install`, not at the top of the formula here.

File: Library/Formula/pidcat.rb [hunk 2]
```diff
@@ -8,6 +9,10 @@ class Pidcat < Formula
 
   def install
     bin.install "pidcat.py" => "pidcat"
+
+    if build.with? "bash-completion"
+      bash_completion.install "bash_completion.d/pidcat"
```
Comment (DomT4) [inline Library/Formula/pidcat.rb]: Likewise, if it needs the dependency make this a single line `if` statement, not a block.

**Comment by paour** [comment]
Sorry, I wasn't aware `bash_completion.install` was available regardless of the bash-completion package. I've updated the pull request.

**Comment by DomT4** [comment]
Thanks! Could you possibly squash your commits into one commit? Our test-bot is having trouble doing so automatically.

**Comment by paour** [comment]
Abandonning

**Comment by paour** [comment]
Created a new pull request: #40565

**Pull request closed without merging**